scripts/proto-update-deps.sh

```diff
@@ -70,16 +65,20 @@ rm -rf 'proto/cosmos'
 curl -f -sSL "$COSMOS_TARBALL_URL" | $tar --exclude='*/third_party' --exclude='*/testutil' "$PROTO_EXPR"
 
 rm -rf 'proto/tendermint'
-curl -f -sSL "$TM_TARBALL_URL" | $tar --exclude='*/third_party' "$PROTO_EXPR"
+curl -f -sSL "$COMETBFT_TARBALL_URL" | $tar --exclude='*/third_party' "$PROTO_EXPR"
 
+ICS23_FILE='proto/cosmos/ics23/v1/proofs.proto'
+curl -f -sSL "$ICS23_PROTO_URL" -o "$ICS23_FILE" --create-dirs
+
+# TODO[1760]: Do we still need this? since confio moved to cosmos?
 ## insert go, java package option into proofs.proto file
 ## Issue link: https://github.com/confio/ics23/issues/32 (instead of a simple sed we need 4 lines cause bsd sed -i is incompatible)
 # See: https://github.com/koalaman/shellcheck/wiki/SC2129
-{
-  head -n3 "$CONFIO_FILE.orig"
-  printf 'option go_package = "github.com/confio/ics23/go";\n'
-  printf 'option java_package = "tech.confio.ics23";\n'
-  printf 'option java_multiple_files = true;\n'
-  tail -n+4 "$CONFIO_FILE.orig"
-} > "$CONFIO_FILE"
-rm "$CONFIO_FILE.orig"
+# {
+#   head -n3 "$ICS23_FILE.orig"
+#   printf 'option go_package = "github.com/confio/ics23/go";\n'
+#   printf 'option java_package = "tech.confio.ics23";\n'
+#   printf 'option java_multiple_files = true;\n'
+#   tail -n+4 "$ICS23_FILE.orig"
+# } > "$ICS23_FILE"
+# rm "$ICS23_FILE.orig"
```

Comment:
The new `proofs.proto` file has the a `go_package` line, so we probably don't need that customization still. It doesn't have the `java_package` or `java_multiple_files` options though. The stuff in the `/protoBindings` directory does some java stuff with all these protos (supposedly), and might be affected.

I'm pretty sure the new `buf` stuff we're switching too allows that to be defined at a higher level (e.g. in a `.buf` file somewhere). If we still need those java options, it'd be great to do it that way (instead of modifying the file like we've been doing).

Reply:
Ok, I will remove this.  Thanks for the input on that.